boskos/janitor/janitor.py

```diff
@@ -49,6 +49,8 @@
     Resource('', 'compute', 'instance-groups', None, 'zone', 'Yes', False, True),
     Resource('', 'compute', 'instance-groups', None, 'zone', 'No', False, True),
     Resource('', 'compute', 'instance-templates', None, None, None, False, True),
+    Resource('', 'compute', 'sole-tenancy', 'node-groups', None, None, False, True),
+    Resource('', 'compute', 'sole-tenancy', 'node-templates', None, None, False, True),
```

Comment:
the 5th arg has a 'zone' or 'region' condition which can be passed into the command

Reply:
Yep that fixed it. I misunderstood the purpose of the arg.